src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfMeansContentProvider.java

```diff
@@ -0,0 +1,199 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.BankAccount;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Child;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.CourtOrder;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Debt;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Employment;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Expense;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Income;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.OnTaxPayments;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Residence;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.SelfEmployment;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.StatementOfMeans;
+
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+
+@Component
+public class StatementOfMeansContentProvider {
+
+    public Map<String, Object> createContent(StatementOfMeans statementOfMeans) {
+        requireNonNull(statementOfMeans);
+
+        ImmutableMap.Builder<String, Object> contentBuilder = ImmutableMap.builder();
+
+        Residence residence = statementOfMeans.getResidence();
+        contentBuilder.put("residence", residence);
+        contentBuilder.put("residenceTypeDescription", residence.getType().getDescription());
```

Comment:
Don't we need to print other residence details if OTHER is selected? Worth asking UX/UC/BA.